tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -951,6 +958,16 @@ def test_positive_get_links(self):
         #          '/katello/api/organizations/:organization_id/content_views',
         #          '/katello/api/organizations/:organization_id/content_views',
         #     ], …}
+        api_paths = response.json()['links']
+        transformed_paths = {ep: tuple(paths.values()) for ep, paths in api_paths.items()}
+        api_diff = DeepDiff(
```

Comment:
Deepdiff covers both assertions that were made before with comparing the keys and values within the groups.

It provides a rich diff when the test case does fail, making it far easier to identify what the difference between expected and transformed data structures are.